apis/v1alpha3/clusterworkloadresourcemapping_types.go

```diff
@@ -18,24 +18,49 @@ package v1alpha3
 
 import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
-// ClusterWorkloadResourceMappingVersion defines the mapping for a specific version of an workload resource.
-type ClusterWorkloadResourceMappingVersion struct {
+// ClusterWorkloadResourceMappingTemplate defines the mapping for a specific version of an workload resource to a
+// logical PodTemplateSpec-like structure.
+type ClusterWorkloadResourceMappingTemplate struct {
 	// Version is the version of the workload resource that this mapping is for.
 	Version string `json:"version"`
```

Comment:
we should define that the version must be non-empty.

Reply:
Feel free to send a PR to the spec.

Reply:
hm, I see that is it required, but I thought that we want to add `+kubebuilder:validation:MinLength=1` on this field.